view/rest/handlers/comment.go

```diff
@@ -1,17 +1,19 @@
 package handlers
 
 import (
-	models2 "github.com/serhiihuberniuk/blog-api/view/rest/models"
 	"net/http"
 
+	models2 "github.com/serhiihuberniuk/blog-api/view/rest/models"
+
 	"github.com/gorilla/mux"
 	"github.com/serhiihuberniuk/blog-api/models"
+	models2 "github.com/serhiihuberniuk/blog-api/view/rest/models"
```

Comment:
do we need two times import the same package? and please use more mean full name then `models2`